README.md

```diff
@@ -1,6 +1,17 @@
-## GenAI Pipeline Repository
+# OpenVINO™ GenAI
```

Comment:
Out of curiosity, is `TM` required? The benefit of not having that is that it's potentially easier to scan the repo for non-ASCII chars. But we don't have it for now, so no need to change.